flow/__init__.py

```diff
@@ -44,10 +46,23 @@
 ]
 
 
-if _get_config_value(
+_import_packaged_environments = _get_config_value(
     "import_packaged_environments",
-    default=_FLOW_CONFIG_DEFAULTS["import_packaged_environments"],
-):
+    default=None,  # Use None to determine if this is being used or not.
+)
+
+if _import_packaged_environments is None:
+    _import_packaged_environments = _FLOW_CONFIG_DEFAULTS[
+        "import_packaged_environments"
+    ]
+else:
+    warnings.warn(
+        "Config key import_packaged_environments will be removed in signac-flow version 0.21. "
+        "Environments will always install once removed. Remove key from config to remove warning.",
```

Comment:
This is worded kind of awkwardly. I suggest the following (line wrapping is left as an exercise to the reader).
```suggestion
        "To remove this warning, remove the flow.import_packaged_environments key from your signac configuration. In the future, environments provided by signac-flow will always be imported."
```